src/generated/resources/schema.graphql

```diff
@@ -80,6 +81,17 @@ type DirectSchemaRegistry {
   uri: String!
 }
 
+type FlinkComputePool {
+  cloud: String
```

Comment:
Also pretty minor, but is this `cloud` (instead of `provider`) for consistency with the CCloud response(s)?